### PR TITLE
Remove trailing spaces throughout

### PR DESCRIPTION
**Pull request by nponeccop**
This happens often in PRs so fixing this everywhere will make many PRs mergeable as they won't include irrelevant whitespace fixes

### Background

This is a focused change to enable linting and other large-scale PRs in the future.

### Changes

Whitespace is eliminated everywhere, results confirmed through git grep. I only changed the space, not double lines of tabs (not sure if they are present).

Some files have a line break added at the end so they are terminated with the line terminator. This is a good practice and it is also enforced by vim, making the repo more vim-friendly.

### Test Plan

The change is supposed to be safe.

### Change Safety

- [ ] I have added tests to cover my changes
- [x] I have considered potential risks and mitigations for my changes

This is a formatting change so it's untestable.

### CODE REVIEW

File: outputs/logs/message-log-1.txt [hunk 1]
```diff
@@ -50,7 +50,7 @@ Gmail: entrepreneurgpt@gmail.com
 Twitter: @En_GPT
 """
 -----------------------------------------------------------------------
-Entrepreneur-GTP: 
+Entrepreneur-GTP:
```
Comment (richbeales) [inline outputs/logs/message-log-1.txt:53]: unrelated, but just noticed this is spelt wrong GTP/GPT

**Review by richbeales — APPROVED** [review]
wide ranging but harmless change.

File: outputs/logs/message-log-1.txt (hunk 1 above)
Comment (nponeccop) [reply outputs/logs/message-log-1.txt:53]: The diff shows It wasn't me.

**Comment by nponeccop** [comment]
Arguably it isn't wide ranged. It's the same trivial thing changed everywhere semi-automatically (actually through a combination of grep and vim s/  *$//). And it improves the overall quality of existing PRs.

File: scripts/json_parser.py [hunk 2]
```diff
@@ -6,8 +6,9 @@
 
 cfg = Config()
 
-JSON_SCHEMA = """
-{
+def fix_and_parse_json(json_str: str, try_to_fix_with_gpt: bool = True):
+    json_schema =
```
Comment (Torantulino) [inline scripts/json_parser.py]: What's up with this diff?

**Review by Torantulino — APPROVED** [review]

File: scripts/json_parser.py (hunk 2 above)
Comment (nponeccop) [reply scripts/json_parser.py]: Conflict resolution went wrong

**Review by Torantulino — CHANGES_REQUESTED** [review]
Looks good expect could you please take a look at the highlighted change above? Something doesn't seem right there

**Comment by Torantulino** [comment]
Cheers

**Review by Torantulino — APPROVED** [review]

**Review by Torantulino — APPROVED** [review]

**Pull request merged**